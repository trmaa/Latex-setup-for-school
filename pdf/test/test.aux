\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Xsonalae Xsonalea}{2}{chapter.1}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}El sexo del Oxxo}{3}{section.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.1}El sexo en exceso}{3}{subsection.1.1.1}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {1.1}{\ignorespaces Moscacaption}}{4}{figure.caption.2}\protected@file@percent }
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:Moscalable}{{1.1}{4}{Moscacaption}{figure.caption.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2}Pene de monicaco}{5}{section.1.2}\protected@file@percent }
\newlabel{LastPage}{{1.2}{6}{Pene de monicaco}{page.6}{}}
\gdef\lastpage@lastpage{6}
\gdef\lastpage@lastpageHy{6}
\gdef \@abspage@last{8}
