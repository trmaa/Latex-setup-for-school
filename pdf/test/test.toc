\contentsline {chapter}{\numberline {1}Xsonalae Xsonalea}{2}{chapter.1}%
\contentsline {section}{\numberline {1.1}El sexo del Oxxo}{3}{section.1.1}%
\contentsline {subsection}{\numberline {1.1.1}El sexo en exceso}{3}{subsection.1.1.1}%
\contentsline {section}{\numberline {1.2}Pene de monicaco}{5}{section.1.2}%
