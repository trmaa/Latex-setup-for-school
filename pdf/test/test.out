\BOOKMARK [0][-]{chapter.1}{\376\377\000X\000s\000o\000n\000a\000l\000a\000e\000\040\000X\000s\000o\000n\000a\000l\000e\000a}{}% 1
\BOOKMARK [1][-]{section.1.1}{\376\377\000E\000l\000\040\000s\000e\000x\000o\000\040\000d\000e\000l\000\040\000O\000x\000x\000o}{chapter.1}% 2
\BOOKMARK [2][-]{subsection.1.1.1}{\376\377\000E\000l\000\040\000s\000e\000x\000o\000\040\000e\000n\000\040\000e\000x\000c\000e\000s\000o}{section.1.1}% 3
\BOOKMARK [1][-]{section.1.2}{\376\377\000P\000e\000n\000e\000\040\000d\000e\000\040\000m\000o\000n\000i\000c\000a\000c\000o}{chapter.1}% 4
